### DISCUSSION TITLE
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'NoneType' object has no attribute 'volume'

### DISCUSSION THREAD

**Discussion by fily-gif** [Rapptz/discord.py#7089] [Q&A] [answered]
help, i dont know how to fix it, i have installed my FFmpeg to path, discord.py 1.0.1 code copied from this repository in examples/baisc_voice.py

**Comment by fily-gif** [comment, marked as answer]
ok nvm fixed

**Reply by maroyanp** [reply]
help me fix mine

**Reply by fily-gif** [reply]
dude, it was almost 1 year ago, i dont remember how i fixed it, search google
